extensions/asset/README.md

```diff
@@ -0,0 +1,31 @@
+# Asset Definition Specification
+
+**Extension [Maturity Classification](../README.md#extension-maturity): Proposal**
+
+A Collection extension to provide details about assets that are available in member Items.
+
+- [Example](examples/example-landsat8.json)
+- [JSON Schema](json-schema/schema.json)
+
+## Collection fields
+
+This extension introduces a single new field, `assets` at the top level of a collection. An Asset Object defined at the Collection level is nearly the same as the [Asset Object in Items]((../../item-spec/item-spec.md#asset-object)), except for two differences. The `href` field is not required, because collections don't point to any data by themselves. Additioanlly the remaining fields, `title` and `type` are required in the Asset Definition, in order for it to adequately describe Item assets.
+
+| Field Name         | Type   | Description                                                  |
+| ------------------ | ------ | ------------------------------------------------------------ |
+| assets | Map<string, Asset Object | **REQUIRED.** A dictionary of assets that can be found in member Items |
+
+### Asset Object
+
+An asset is an object that contains details about the datafiles that will be included in member Items. Assets included at the Collection level do not imply that all assets are available from all Items. However, it is recommended that the Asset Definition is a complete set of all assets that may be available from any member Items.
+
+| Field Name | Type   | Description                                                                           |
+| ---------- | ------ | ------------------------------------------------------------------------------------- |
+| title      | string | **REQUIRED.** The displayed title for clients and users.                                            |
```

Comment:
A title is not required for assets in Items. So that's a bit contradicting... What should I do with assets that have no title?

Reply:
Well it's not contradicting in that this Asset object is different. It does not require href (in fact that is not a field), and does require title and type.

The reasoning here is that if an "Asset Definition" is being provided than there should be some sort of label/description of each of the assets.

Reply:
I still think it's a problem to require it here. A script that crawls Items and updated the Collection schemas automatically would fail to comply. It either doesn't have a title, leave the schema out or have to invent one. All three options are not good.

Reply:
Well, you shouldn't be creating the Collections schema from Items I don't think - what if you only have href defined?  You'll have no asset definition. Or are you ok with requiring type in this extension, in which case how is that different?

Reply:
Oh, hell... When I was checking the Item spec on my phone yesterday night, I mixed up href and type and thought type would be required. You are right that type is not required, too. Well, in this case I wouldn't require anything, but I see that having empty objects doesn't help anybody. Hmm... I'll just leave this to others to decide, I think. ;-)

Reply:
So isn't using this extension basically requiring titles and types in Items as the extension also says that all assets should be listed?

Reply:
Not sure if I understand what you are saying here.

This extension doesn't apply to the Item at all, so you can do whatever you want there.  

You also don't need to include *all* assets, although it is recommended. 

The idea here is that if your collection is going to have a definition of the assets it should probably include some minimum amount of information (that is a short description of the asset via `title`, and the type of file via `type`). If those things aren't present, or only one of them is, it's not that useful. 

Users should be able to use the Asset Definition to determine what assets they are interested in accessing.

Reply:
I think perhaps you need to explain more what this 'title' is for, that it is a short description of the asset for users to determine which assets they are interested in accessing. And maybe provide some more information on use than that. The 'The displayed title for clients and users.' doesn't really help people figure out what they should put here.